string shipment_type
---
int64 priority
